src/backend/metal/src/lib.rs

```diff
@@ -65,6 +66,7 @@ impl Shared {
     pub(crate) fn new(device: metal::Device) -> Self {
         Shared {
             queue_pool: Mutex::new(command::QueuePool::default()),
+            aux_queue: Mutex::new(device.new_command_queue()),
```

Comment:
What happens if we exceed `64` temporary command buffers?

Reply:
I don't think this is a problem, since we commit those command buffers right after acquiring them. So in the worst case there is going to be a bit of a delay for GPU to finish.

Reply:
I guess it depends on how many active deferred commands buffers we actually expect in real use cases, i.e. if it becomes hundreds in practice it would be unfortunate to hit that delay often IMO

Reply:
Hmm. Theoretically, it can become an issue, but it's too early to worry about this right now